### PR TITLE
:sparkles: Add Design for Landing Page

### PR DESCRIPTION
**Pull request by welschoff**
Add Design for Landing Page

Screenshot 2021-11-03 at 14 34 21

### CODE REVIEW

File: src/app/App.module.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  grid-template-rows: 30% 1fr;
+  background: rgb(128 128 128 / 38.6%);
+  border-radius: 10px;
+  width: 50%;
+  height: 50%;
+  place-items: center;
+  overflow: hidden;
+}
+
+.submit:hover {
```
Comment (fabiglahn) [inline src/app/App.module.css:37]: Cool addition for you hover: duration
```
.submit {
transition-duration: 0.5s;
}
```

File: src/app/components/Title/Title.module.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.title {
+  display: grid;
+  place-items: center;
```
Comment (fabiglahn) [inline src/app/components/Title/Title.module.css:3]: I think you just need the 'text-align: center'

**Review by fabiglahn — COMMENTED** [review]
Looks really good! Friday: 🍻

File: src/app/App.module.css (hunk 1 above)
Comment (welschoff) [reply src/app/App.module.css:37]: Thanks :)

**Pull request merged**